styles/08-midnight.json

```diff
@@ -218,6 +202,9 @@
 			},
 			"core/quote": {
 				"typography": {
+					"fontSize": "var:preset|font-size|medium",
+					"letterSpacing": "-0,01em",
```

Comment:
Same here, change out the comma for the period.

Reply:
Actually, same here as with the buttons: Values that are identical to do those on the body does not need to be duplicated on the block.